src/bl/boardFacade.ts

```diff
@@ -66,12 +68,14 @@ export class BoardFacade {
         }
     }
 
-    static async update(params: {}, boardId: number): Promise<Board> {
+    static async update(params: express.Request, boardId: number): Promise<Board> {
```

Comment:
Facade should not get any express stuff. You need to give the the body and the requester.
`static async update(requester: Requester, boardId: number, params: {}): Promise<Board> {`